src/workspaces/helpers/database.tsx

```diff
@@ -0,0 +1,45 @@
+type lang = "R" | "PYTHON";
+
+const trimText = (text: string): string => {
```

Comment:
What if there is a correct indent ?
It's better to write the snippet with the correct indent directly than trying to massage it